src/main/java/uk/gov/hmcts/reform/sscs/helper/ResourceLoader.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class ResourceLoader {
+
+    private ResourceLoader() {
+    }
+
+    public static String loadJson(final String filePath) throws URISyntaxException, IOException {
+        return new String(loadResource(filePath), StandardCharsets.UTF_8);
+    }
+
+    private static byte[] loadResource(final String filePath) throws URISyntaxException, IOException {
```

Comment:
I would have thought there is a library for this

Reply:
ResourceLoader removed